features/support/hooks.rb

```diff
@@ -8,5 +8,6 @@
 end
 
 Before ('@javascript') do
-  Metric.create(name: "dashboard-0", time: "2017-08-03T15:00:00+00:00")
+  Metric.create(name: "dashboard-0")
+  # TODO - is there a better way to accomplish this
```

Comment:
presumably this is setup for the dashboard tests? It would be better in a separate before block for just the tests involved, or use a different tag, rather than add onto the @javascript tag which already has meaning.